statefun-e2e-tests/statefun-smoke-e2e-java/src/main/java/org/apache/flink/statefun/e2e/smoke/Constants.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.flink.statefun.e2e.smoke;
+
+import org.apache.flink.statefun.e2e.smoke.generated.Command;
+import org.apache.flink.statefun.e2e.smoke.generated.Commands;
+import org.apache.flink.statefun.e2e.smoke.generated.SourceCommand;
+import org.apache.flink.statefun.e2e.smoke.generated.VerificationResult;
+import org.apache.flink.statefun.sdk.java.TypeName;
+import org.apache.flink.statefun.sdk.java.types.SimpleType;
+import org.apache.flink.statefun.sdk.java.types.Type;
+
+final class Constants {
+  private Constants() {}
+
+  private static final String APP_NAMESPACE = "statefun.smoke.e2e";
+  private static final String PROTO_TYPES_NAMESPACE = "type.googleapis.com";
```

Comment:
I don't think we need to strictly use this namespace for user-level messages.
The only reason this was required was because on the driver side, you're using the `TypedValueUtils` to unwrap / wrap Protobuf messages from / to `TypedValue`s.

Since `SourceCommand`, `Commands`, and `VerificationResult` is pure user-space messages, lets reflect that by using custom typenames.

Reply:
OK. I think what you're saying is TypedValueUtils should be used only for Statefun internal processing instead of user level code. Am I getting it right?